package.json

```diff
@@ -24,7 +24,10 @@
     "node": ">=0.10"
   },
   "scripts": {
-    "install": "prebuild --install",
+    "build:libzmq": "node scripts/download-win-lib.js && ./build_libzmq.sh",
```

Comment:
How can I prevent `./build_libzmq.sh` from running on Windows or do I have to use a node script?